index.js

```diff
@@ -13,6 +13,17 @@ function hex(name) {
 	return htmlColors[name.toLowerCase()];
 }
 
+function rgb(name) {
+	if (typeof name !== 'string') {
+		throw new TypeError('htmlColors.hex expected a string, got ' + typeof name);
+	}
+	const hex = htmlColors[name.toLowerCase()];
```

Comment:
Maybe a useful check or error here in case the supplied `name` is not a valid html color value